testsuite/features/reposync/allcli_update_activationkeys.feature

```diff
@@ -224,5 +244,6 @@ Feature: Update activation keys
     And I follow "Terminal Key x86_64" in the content area
     And I wait for child channels to appear
     And I check "Fake-RPM-Terminal-Channel"
+    And I wait until "Fake-RPM-SUSE-Channel" has been checked
```

Comment:
Is this correct or should it match "Fake-RPM-Terminal-Channel" ?

Reply:
Thank you. This was a copy-paste error. Fixeda and since this was only a small fix, I amended the changes to the same commit.